fluid_dynamics/validation/two_fluid_wave/source/ProjectParameters.json

```diff
@@ -53,7 +53,10 @@
             "block_size" : 3,
             "use_block_matrices_if_possible" : true,
             "coarse_enough" : 500
-    	}
+    	},
+        "distance_modification_settings": {
+            "distance_threshold": 0.0000001
```

Comment:
I think it is better to write it in the scientific format as "1.0e-7" to be better read.

Reply:
Agree. I just copied and pasted the old values. Updating it.